data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+			`	He asks a few questions for clarity and when you're done he requests if you'd kindly wait for a moment while he makes a call. It is a lengthy wait, and at one point an aide comes in and offers you refreshments while deliberations continue elsewhere. Much to your surprise the fruity beverage is actually quite light and not too sweet, reminding you briefly that this is actually the world where your major foods originally evolved.`
+			`	Eventually Mr Yang returns, "So sorry for that, I hope you were looked after suitably while you waited."`
+			`	Once you nod your assent he continues, "Well, obviously the Republic knows about the Hai, but for the moment our official line is that the government does not trade in rumours. However, clearly an appropriate response is needed. I have been informed that Republic Intelligence has gone through official channels to brief the Free Worlds Council just yesterday. They have demanded that they have a trained diplomatic representative along for any decisions. This is already standard practice for the Syndicate. If you are willing, I will come with you, and we shall meet two individuals known as Katya and Alastair on Deep and Foundry respectively. I believe you already know them?"`
+			`	You indicate as such.`
+			`	"Excellent, in that case we should go to Deep first for speed, shall we be off?"`
+				accept
+	on visit
+		dialog phrase "generic stopover on visit"
+	on complete
+		log `You have brought a number of diplomatic representatives from each of the main human factions to Hai-home to attend to the crisis in a collaborative manner. They do not believe any of their automatic responses that have already been engaged will be enough to suppress the information leak, though there is a reasonable model for how long they have to work with. While they work on the problem though you have been asked to help route some pirates. Apparently less scrupulous individuals have reacted swiftly to the leak and investigated in the hopes of scoring quickly.`
+		conversation
+			`You are directed to an official landing pad on your return to Hai-home and it is not long before you are introducing your entourage to Sayari. Alondo in particular seems quite stoked to be here and seems to have a knack for charming Sayari with polite and insightful, if rather simple, questions throughout the diplomatic niceties.`
+			`	You get down to business at the moment Sayari inquires how your journey went.`
+			choice
+				`	"I don't recall there being nearly so many pirates."`
+			`	Sayari frowns and exhales a long breath through her nose, "Yes. Well it seems that the rumours of us, and the information making its way through your media has attracted every opportunist this side of the Deep."`
+			`	"Of course," she says, "some of the pirates were aware of us, and we were aware of them being aware of us. We had a quiet arrangement with a select few where we rewarded them for keeping the secret well contained within their own circles."`
```

Comment:
Because what follows is a qualifying statement I wanted to give it a bit of distance stylistically from the previous one and this seemed the smoothest way to do that. Were you just removing it for brevity?

Reply:
Yes, that was my intent. But your point is a good reason to leave it in, so feel free to resolve.